nimbus/src/error.rs

```diff
@@ -15,6 +15,8 @@ pub enum Error {
     RkvError(rkv::StoreError),
     #[error("IO error: {0}")]
     IOError(#[from] std::io::Error),
+    #[error("Number parsing error: {0}")]
```

Comment:
I'm not sure it's worth exposing this to consumers - I'd suggest just ignoring the header if it is malformed in this way.